dataactbroker/README.md

```diff
@@ -545,15 +545,91 @@ Example output:
   "job_3012_error_url": "https...",
   "job_3006_error_url": "https...",
   "job_3010_error_url": "https...",
-  "job_3008_error_url": "https..."
+  "job_3008_error_url": "https...",
+  "cross_appropriations-program_activity": "https..."
```

Comment:
I clarified the text in process overview and specified that the length rule is a warning in available validations.  The available validations section is actually obsolete now as it's referring to the rule types we have deleted.  I'm thinking we should delete this section, but we may want to preserve the into on type, required, and length checks in some form.